### ISSUE TITLE
Run SonarTsPlugin AND SonarSource/SonarTS 

### ISSUE THREAD

**Issue by guillaume317** [Pablissimo/SonarTsPlugin#182]
Hi Team,

Is it possible to run together on a same instance your plugin and the new SonarSource/SonarTS one ?
Both plugins have the same name. 
When I try to rename it, another problem occurs with the following message:
"java.lang.IllegalArgumentException: Multiple entries with same key: ts=TypeScript and ts=TypeScript "

**Comment by nixel2007** [comment]
No, you can't. Even if you handle the same key issue, you'll face the same extension error - only one plugin can be registred to analyze ts extension. To make work both plugins, this plugin must be re-written to "custom rules plugin" api, extending the official one.

**Comment by philipooo** [comment]
Would be nice if we could use this plugin side by side. I love the way how tslint-output can be imported with the help of this plugin. The original SonarQube plugin has no way doing this.

**Comment by ClaudioLelis** [comment]
Now in SonarQube 7.4 seems like SonarTSPlugin does not working. Is that true? I uninstalled the SonarSource/SonarTS and even not worked.

**Comment by philipooo** [comment]
Btw SonarSource/SonarTS now supports import of tslint output.

**Comment by ClaudioLelis** [comment]
Yes, but I can not automatically run TSLint through sonar-scanner, and can not manage TSLint core and custom rules, right?
